docs/source/issues.rst

```diff
@@ -55,7 +59,18 @@ ElasticBLAST uses cloud storage to temporally store query sequences and internal
    gsutil -m rm gs://${ELB_RESULTS_BUCKET}/logs/*  # to delete metadata files
 
 
+.. _early_shutdown:
+
 A synchronous search may shut down too early
 --------------------------------------------
 
 When doing a synchronous search (submitted with the ``--sync`` option), ElasticBLAST is continuously probing for search status to know when the search is done. When the status check times out ElasticBLAST interprets it as search failure and shuts down the cluster.
+
+.. _too_many_jobs:
+
+Too many jobs leads to a failed execution
+-----------------------------------------
+
+If the ElasticBLAST configuration leads to too many (e.g.: over 5,000)
```

Comment:
This description seems vague. Do we know that there is a maximum number of jobs a kubernetes cluster can handle? If yes, then we should say that this is the maximum.

Reply:
Yes, see https://cloud.google.com/kubernetes-engine/quotas

Reply:
Yes, this is vague too. The only applicable quota from this page seems to be 300k container per cluster. We have 3 containers per job, which gives us a lot of room. Running this:
```
kubectl get resourcequota gke-resource-quotas -o yaml
```
There is this:
```
spec:
  hard:
    count/ingresses.extensions: "100"
    count/jobs.batch: 5k
    pods: "1500"
    services: "500"
status:
  hard:
    count/ingresses.extensions: "100"
    count/jobs.batch: 5k
    pods: "1500"
    services: "500"
  used:
    count/ingresses.extensions: "0"
    count/jobs.batch: "17"
    pods: "6"
    services: "1"
```
This is probably where maximum of 5k jobs comes from.

Reply:
>This is probably where maximum of 5k jobs comes from.

Yes.